rce/src/job/files.ts

```diff
@@ -0,0 +1,15 @@
+export class Files {
+  public readonly files: Array<{ fileName: string, code: string, entrypoint: boolean }>;
```

Comment:
mending pisahin aja ini biar gampang bacanya

```suggestion
type File = {
    fileName: string;
    code: string;
    entrypoint: boolean;
}

export class Files {
  public readonly files: File[];
```